apis/compute/v1beta1/computeforwardingrule_types.go

```diff
@@ -31,7 +31,7 @@ var (
 	// AddToScheme is a global function that registers this API group & version to a scheme
 	AddToScheme = SchemeBuilder.AddToScheme
 
-	ComputeForwardingRuleGVK = schema.GroupVersionKind{
+	GroupVersionKind = schema.GroupVersionKind{
```

Comment:
This is going to cause conflicts - we have multiple GVKs in this package I believe.